spec/api/v1/advocacy_campaigns_spec.rb

```diff
@@ -33,6 +33,21 @@
         expect(api_advocacy_campaign).to eq(serialized_advocacy_campaign)
       end
     end
+
+    describe "GET /v1/advocacy_campaigns?filter[origin_system]" do
+      it "filters advocacy campaigns by origin_system" do
+        FiveCalls_event = create(:advocacy_campaign, title: "Event1", description: "Description1" , origin_system: "5calls", action_type: "phone")
```

Comment:
Can you change `FiveCalls_event` to `fiveCalls_event`?  The interpreter is going to see that capital F and think that you're declaring a constant.

Reply:
Also, can you remove your references to events here, since this file only deals with advocacy_campaigns?